.github/workflows/issue_bot.yml

```diff
@@ -61,5 +75,5 @@ jobs:
               "owner": context.repo.owner,
               "repo": context.repo.repo,
               "issue_number": context.payload.issue.number,
-              "labels": component_labels,
+              "labels": component_labels.concat(non_component_labels),
```

Comment:
can `non_component_labels` be null or empty? what happens on that case?

Reply:
It can be empty. In that case it will process the component labels correctly. I don't think the rest api will ever return nulls. But I'm not sure.

Reply:
Cool, to be honest currently we always add the `Type: xxx` when creating an issue.